Tests/AblyChatTests/IntegrationTests.swift

```diff
@@ -131,64 +134,64 @@ struct IntegrationTests {
+        // (22) Send `.enter` presence event with custom data on our client and check that we receive it on the subscription
         try await txRoom.presence.enter(data: .init(userCustomData: ["randomData": .string("randomValue")]))
         let rxPresenceEnterRxEvent = try #require(await rxPresenceSubscription.first { _ in true })
         #expect(rxPresenceEnterRxEvent.action == .enter)
         #expect(rxPresenceEnterRxEvent.data?.userCustomData?["randomData"]?.value as? String == "randomValue")
 
-        // (22) Send `.update` presence event with custom data on our client and check that we receive it on the subscription
+        // (23) Send `.update` presence event with custom data on our client and check that we receive it on the subscription
         try await txRoom.presence.update(data: .init(userCustomData: ["randomData": .string("randomValue")]))
         let rxPresenceUpdateRxEvent = try #require(await rxPresenceSubscription.first { _ in true })
         #expect(rxPresenceUpdateRxEvent.action == .update)
         #expect(rxPresenceUpdateRxEvent.data?.userCustomData?["randomData"]?.value as? String == "randomValue")
 
-        // (23) Send `.leave` presence event with custom data on our client and check that we receive it on the subscription
+        // (24) Send `.leave` presence event with custom data on our client and check that we receive it on the subscription
         try await txRoom.presence.leave(data: .init(userCustomData: ["randomData": .string("randomValue")]))
         let rxPresenceLeaveRxEvent = try #require(await rxPresenceSubscription.first { _ in true })
         #expect(rxPresenceLeaveRxEvent.action == .leave)
         #expect(rxPresenceLeaveRxEvent.data?.userCustomData?["randomData"]?.value as? String == "randomValue")
 
         // MARK: - Detach
 
-        // (24) Detach the room
+        // (25) Detach the room
         try await rxRoom.detach()
 
-        // (25) Check that we received a DETACHED status change as a result of detaching the room
+        // (26) Check that we received a DETACHED status change as a result of detaching the room
         _ = try #require(await rxRoomStatusSubscription.first { $0.current == .detached })
         #expect(await rxRoom.status == .detached)
 
         // MARK: - Release
 
-        // (26) Release the room
+        // (27) Release the room
         try await rxClient.rooms.release(roomID: roomID)
 
-        // (27) Check that we received a RELEASED status change as a result of releasing the room
+        // (28) Check that we received a RELEASED status change as a result of releasing the room
         _ = try #require(await rxRoomStatusSubscription.first { $0.current == .released })
         #expect(await rxRoom.status == .released)
 
-        // (28) Fetch the room we just released and check it’s a new object
+        // (29) Fetch the room we just released and check it’s a new object
```

Comment:
I don't think it's a good idea to keep these numbers here.

Reply:
There's at least one place where I refer to one step in another one; we need to still be able to do that

Reply:
Maybe to split to a number of tests?

Reply:
I'll do it as part of my next PR